Sources/SwiftSyntaxBuilder/generated/BuildableBaseProtocols.swift

```diff
@@ -0,0 +1,217 @@
+
+//// Automatically Generated by SwiftSyntaxBuilderGeneration
+//// Do Not Edit Directly!
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2022 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import SwiftSyntax
+public protocol DeclListBuildable: SyntaxListBuildable{
```

Comment:
Please rebase on top of the latest `main` and regenerate this file. It should add a space before left braces
```suggestion
public protocol DeclListBuildable: SyntaxListBuildable {
```

Reply:
Done!